I took the time to rebuild llvm-mos's copy of coremark _without this patch_, and we got two warnings. Accordingly, I think there's another place to fix this at line 367.

```
core_main.c:367:42: warning: format specifies type 'int' but the argument has type 'secs_ret' (aka 'unsigned long') [-Wformat]
  367 |     ee_printf("Total time (secs): %d\n", time_in_secs(total_time));
      |                                   ~~     ^~~~~~~~~~~~~~~~~~~~~~~~
      |                                   %lu
core_main.c:370:19: warning: format specifies type 'int' but the argument has type 'ee_u32' (aka 'unsigned long') [-Wformat]
  369 |         ee_printf("Iterations/Sec   : %d\n",
      |                                       ~~
      |                                       %lu
  370 |                   default_num_contexts * results[0].iterations
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  371 |                       / time_in_secs(total_time));
      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
```

--- a/core_main.c
+++ b/core_main.c
@@ -364,7 +364,7 @@ for (i = 0; i < MULTITHREAD; i++)
                   default_num_contexts * results[0].iterations
                       / time_in_secs(total_time));
 #else
-    ee_printf("Total time (secs): %d\n", time_in_secs(total_time));
+    ee_printf("Total time (secs): %"PRIu32"\n", time_in_secs(total_time));
     if (time_in_secs(total_time) > 0)
         ee_printf("Iterations/Sec   : %"PRIu32"\n",
                   default_num_contexts * results[0].iterations

